src/sites/doc/docs/taro/theme-react.md

```
@@ -0,0 +1,53 @@
+
+## 介绍
+
+NutUI-React 支持灵活的样式定制，满足多种视觉业务和品牌需求，包括但不限于全局主色调和特定组件视觉定制的支持。
+<br />
+在 NutUI-React 的 1.4.0 版本中，依然可以使用 1.x 的 Sass 定制主题功能，我们在原有的主题定制功能上新增了 CSS 变量， 无需引入额外的 SCSS 样式文件，就可以使用到新的主题定制功能。
+
+## 方式一 使用 CSS Variables 进行主题配置
+
+`NutUI-React` 组件可以通过 [CSS Vars](https://developer.mozilla.org/zh-CN/docs/Web/CSS/Using_CSS_custom_properties) 来组织样式，通过覆盖这些 `CSS` 变量，可以实现定制主题、动态切换主题等功能。
+<br />
+您可在 [ConfigProvider 组件](#/zh-CN/component/configprovider) 进行体验。
```

Markdownlint (0.37.0)

Link fragments should be valid

(MD051, link-fragments)